apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/DataTransferManagementReceiverImpl.java

```diff
@@ -0,0 +1,211 @@
+package com.apollocurrency.aplwallet.apl.core.shard;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.sql.Connection;
+import java.text.SimpleDateFormat;
+import java.util.Map;
+import java.util.Objects;
+
+import com.apollocurrency.aplwallet.apl.core.app.Block;
+import com.apollocurrency.aplwallet.apl.core.app.Blockchain;
+import com.apollocurrency.aplwallet.apl.core.app.DatabaseManager;
+import com.apollocurrency.aplwallet.apl.core.db.TransactionalDataSource;
+import com.apollocurrency.aplwallet.apl.core.db.model.OptionDAO;
+import com.apollocurrency.aplwallet.apl.util.injectable.DbProperties;
+import org.slf4j.Logger;
+
+/**
+ * {@inheritDoc}
+ */
+@Singleton
+public class DataTransferManagementReceiverImpl implements DataTransferManagementReceiver {
+    private static final Logger log = getLogger(DataTransferManagementReceiverImpl.class);
+    private final SqlSelectAndInsertHelper sqlSelectAndInsertHelper = new SqlSelectAndInsertHelper();
+
+    private MigrateState state = MigrateState.INIT;
+    private DbProperties dbProperties;
+    private DatabaseManager databaseManager;
+    private Blockchain blockchain;
+    private OptionDAO optionDAO;
+    private static final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
```

Comment:
Old dateTimeFormatter, consider java 8 DateTimeFormatter

Reply:
Removed because it's not needed now.